pom.xml

```diff
@@ -583,6 +583,8 @@
 						<target>${javaCompilerVersion}</target>
 						<source>${javaCompilerVersion}</source>
 						<encoding>UTF-8</encoding>
+						<fork>true</fork>
+						<useIncrementalCompilation>true</useIncrementalCompilation>
```

Comment:
@hongdongni  and the second thing  is to confirm whether maven doesnt  set useIncrementalCompilation  to true by default

Reply:
I'm going to delete this line